dataprofiler/profilers/profile_builder.py

```diff
@@ -1112,6 +1114,19 @@ def _save_helper(self, filepath: str | None, data_dict: dict) -> None:
         # Restore all data labelers
         self._restore_data_labelers(data_labelers)
 
+    def _json_save_helper(self, filepath: str) -> None:
```

Comment:
if we decide to keep this format and we allow `filepath` to be `None`, this typing needs to have a `| None`